--- interchange.cfg.dist.orig	Tue Dec 10 23:01:42 2002
+++ interchange.cfg.dist	Thu Jan  8 08:36:28 2004
@@ -46,8 +46,8 @@
 
 ifdef @DEBUG
 
-Message Sending debug to /tmp/icdebug.
-DebugFile /tmp/icdebug
+Message Sending debug to /var/log/interchange/debug.log.
+DebugFile /var/log/interchange/debug.log
 
 # A few simple tests ...
 GlobalSub sub test_global_sub { return 'Test of global subroutine OK.' }
